src/main/resources/db/changelog/changes/05-insert-users-table.yaml

```diff
@@ -7,13 +7,13 @@ databaseChangeLog:
             tableName: users
             columns:
               - column: {name: "email", value: "admin@admin.com"}
-              - column: {name: "password", value: "$2a$10$bbO1IRnLAo5zdo/ayMtrAOAf8f6p.80ZgMvg0yHklt2j4RFwskNI."}
+              - column: {name: "password", value: "$2a$10$lQX.zYRJUWqlYrO8DRd7HeSlvlowl4toXBrk5U9laHlVKQmll7hRS"}
               - column: {name: "first_name", value: "Admin"}
               - column: {name: "last_name", value: "Adminov"}
         - insert:
             tableName: users
             columns:
               - column: { name: "email", value: "user@user.com" }
-              - column: { name: "password", value: "$2a$10$bbO1IRnLAo5zdo/ayMtrAOAf8f6p.80ZgMvg0yHklt2j4RFwskNI." }
+              - column: { name: "password", value: "$2a$10$lQX.zYRJUWqlYrO8DRd7HeSlvlowl4toXBrk5U9laHlVKQmll7hRS" }
```

Comment:
add password in plaintext as comment next to bcrypt cipher, it's useless otherwise